client-js/app/elements/dir-item.html

```diff
@@ -0,0 +1,30 @@
+
+<dom-module id='dir-item'>
+  <template>
+    <paper-icon-item><iron-icon icon="{{dir_icon}}" item-icon></iron-icon><content></content></paper-icon-item>
+
+
+  </template>
+
+  <script>
+    Polymer({
+        is:"dir-item",
+        properties: {
+            dir_name: {type: String},
+            prop: {type: String},
```

Comment:
Yes, thanks for picking that up this is a vestige from when I was trying to learn polymer.